LICENSE.md

```diff
@@ -73,19 +79,19 @@ The User may examine the Mod's source code.
 The Author reserves the right to distribute the Mod.  
 The Author may designate a person or organization as a Distributor.  
 A Distributor is granted the right to distribute the Mod.  
-The Author may require that a Distributor provides a direct link to the Codebase.  
+The Author may require that a Distributor provides a direct link to the Mod.  
 The User may not distribute the Mod without being granted distribution rights from the Author.  
-The Author may revoke distribution rights from any Distributor.  
+The Author may revoke distribution rights from any Distributor without prior notice.  
```

Comment:
Unnecessary. Nowhere does this license say anything about giving notice. No need to start mentioning it now.